CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Main (unreleased)
 
+- [FEATURE] Put Tests requiring Network Access behind a -online flag (@flokli)
```

Comment:
Can you add this to the end of the features? I tend to keep them sorted by type and then date.

Reply:
Sure! Wasn't aware it was sorted this way. Updated.